setup.cfg

```
@@ -82,6 +82,8 @@ console_scripts =
 connector_matrix =
 	bleach>=3.1.5
 	matrix-nio>=0.14.1
+	python-olm
+	matrix-nio[e2e]
```

I don't think we want this here like this.

There are two things, I think `python-olm` should be a dependency of `matrix-nio[e2e]` anyway, also I think that we don't want to require e2e for installing the matrix connector, I think it needs to be opt-in as it's difficult to install on non-linux platforms at the moment.

What I propose is this:
```suggestion
connector_matrix_e2e =
	matrix-nio[e2e]
```